integrations/zookeeper/documentation.yaml

```diff
@@ -1,6 +1,6 @@
 exporter_type: included
 app_name_short: Zookeeper
-app_name: {{app_name_short}}
+app_name: Apache  {{app_name_short}}
```

Comment:
There's an extra whitespace here between Apache and the short name. Please fix before merging.